podpac/core/data/types.py

```diff
@@ -694,7 +697,165 @@ def _find_h5py_keys(obj, keys=[]):
         keys = list(set(keys))
         keys.sort()
         return keys
-            
+
+class Zarr(DataSource):
+    source = tl.Unicode(allow_none=True)
```

Comment:
should `source` not be tagged? Even if it could be None? 

Why could source be None?

Reply:
oh wait... it's a datasource node, so `source` is always automatically tagged.